### DISCUSSION TITLE
Deploying code to elasticbeanstalk with docker

### DISCUSSION THREAD

**Discussion by Julioliveira** [prisma/prisma#6559] [Q&A] [answered]
Hey, everyone!
I'm trying to deploy my application to EB but I cannot quite do it.
I'm using express, TS and prisma. But when my dockerfile tries running prisma generate, it generates an error.

I have the following files:

DOCKERFILE:
`FROM node:lts-alpine

RUN mkdir -p /home/node/api/node_modules && chown -R node:node /home/node/api

WORKDIR /home/node/api

COPY --chown=node:node package.json ./

RUN yarn global add pm2 typescript prisma

USER node

COPY --chown=node:node . .

RUN yarn

RUN tsc

EXPOSE 8080

CMD ["pm2-docker", "process.json"]
`

docker-compose.yml

`version: "3"
services:
  app:
    container_name: app
    restart: always
    build: .
    ports:
      - "8080:8080"
    volumes:
      - ./:/app
      - /app/node_modules
`

Dockerrun.aws.json

`{
  "AWSEBDockerrunVersion": 1,
  "volumes": [
    {
      "name": "prontuario",
      "host": {
        "sourcePath": "/var/app/prontuario"
      }
    }
  ],
  "containerDefinitions": [
    {
      "name": "prontuario",
      "image": "node:lts-alpine",
      "memory": 128,
      "portMappings": [
        {
          "hostPort": 8080,
          "containerPort": 8080
        }
      ],
      "mountPoints": [
        {
          "sourceVolume": "prontuario",
          "containerPath": "/app"
        }
      ]
    }
  ]
}
`

Does anyone know how to make ir work?

**Comment by ryands17** [comment, marked as answer]
@Julioliveira 👋 

Have a look at [this](https://github.com/ryands17/nexus-auth/blob/master/Dockerfile) that deploys to Beanstalk on the Docker environment. Also remove your `docker-compose.yml` as that sometimes messes up with the deployment.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
